nixos/modules/security/acme.nix

```diff
@@ -336,7 +338,10 @@ in
                           chmod ${fileMode} *.pem
                           chown '${data.user}:${data.group}' *.pem
 
-                          ${data.postRun}
+                          if [ "$KEY_CHANGED" = "yes" ]; then
+                            : # noop in case postRun is empty
```

Comment:
is this `:` a typo or is this a `bash-ism` i dont know?

Reply:
It’s a bash-ism you don’t know :)
It’s `noop`

Reply:
I may replace it with a better-known `true` if you’re more comfortable

Reply:
It's fine